### PR TITLE
[opentitantool] Fix CRC computation for alert_handler

### PR DESCRIPTION
**Pull request by jon-flatley**
Register values for enabled alert classes were being set incorrectly due to the order of a fallthrough switch case in ROM being read in reverse.

### CODE REVIEW

**Comment by jon-flatley** [comment]
This PR fixes an issue where ROM would fail with `kErrorAlertBadCrc32` when trying to boot with any byte of `ROM_ALERT_CLASS_EN` set to `kAlertEnableEnabled` or `kAlertEnableLocked` when using the `otp_alert_digest` bazel rule to generate the alert digest OTP values.

File: sw/host/opentitanlib/src/otp/alert_handler.rs [hunk 1]
```diff
@@ -275,10 +275,10 @@ impl AlertRegs {
         match config.enabled {
             AlertEnable::None => {}
             AlertEnable::Enabled => {
-                reg_bits.set(ALERT_HANDLER_CLASSA_CTRL_SHADOWED_LOCK_BIT as usize, true);
                 reg_bits.set(ALERT_HANDLER_CLASSA_CTRL_SHADOWED_EN_BIT as usize, true);
             }
             AlertEnable::Locked => {
+                reg_bits.set(ALERT_HANDLER_CLASSA_CTRL_SHADOWED_LOCK_BIT as usize, true);
```
Comment (alphan) [inline sw/host/opentitanlib/src/otp/alert_handler.rs:281]: For reference, [this](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/drivers/alert.c;drc=b4d0de8f261f6397c4af74d33b0dc7afae73e224;l=169) is the corresponding line from the alert handler driver.

**Review by alphan — APPROVED** [review]
Thanks @jon-flatley !

**Comment by alphan** [comment]
This is not urgent at all, but it would be great if you could create a gist for the helper tool that you wrote and add its link to this PR.

**Pull request merged**